packages/mask/src/social-network-adaptor/facebook.com/collecting/posts.tsx

```diff
@@ -110,8 +105,6 @@ function collectPostsFacebookInner(store: Next.CollectingCapabilities.PostsProvi
                     info.postMetadataImages.add(url)
                     nextTypedMessage.push(makeTypedMessageImage(url))
                 }
-                // parse post content
-                info.postMessage.value = makeTypedMessageTuple(nextTypedMessage)
```

Comment:
is this ok?

Reply:
<img width="761" alt="image" src="https://user-images.githubusercontent.com/72531217/203236351-426dd02f-be3f-41e9-b440-b757eae300b5.png">